tests/helper/helper_configValidate.go

```diff
@@ -0,0 +1,138 @@
+package helper
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"reflect"
+	"strconv"
+	"strings"
+
+	. "github.com/onsi/gomega"
+	"gopkg.in/yaml.v2"
+)
+
+const configFileDirectory = ".odo"
+const configFileName = "config.yaml"
+
+type config struct {
+	ComponentSettings struct {
+		Type           string   `yaml:"Type,omitempty"`
+		SourceLocation string   `yaml:"SourceLocation,omitempty"`
+		Ref            string   `yaml:"Ref,omitempty"`
+		SourceType     string   `yaml:"SourceType,omitempty"`
+		Ports          []string `yaml:"Ports,omitempty"`
+		Application    string   `yaml:"Application,omitempty"`
+		Project        string   `yaml:"Project,omitempty"`
+		Name           string   `yaml:"Name,omitempty"`
+		MinMemory      string   `yaml:"MinMemory,omitempty"`
+		MaxMemory      string   `yaml:"MaxMemory,omitempty"`
+		DebugPort      []int    `yaml:"DebugPort,omitempty"`
+		Storage        []struct {
+			Name string `yaml:"Name,omitempty"`
+			Size string `yaml:"Size,omitempty"`
+			Path string `yaml:"Path,omitempty"`
+		} `yaml:"Storage,omitempty"`
+		Ignore bool   `yaml:"Ignore,omitempty"`
+		MinCPU string `yaml:"MinCPU,omitempty"`
+		MaxCPU string `yaml:"MaxCPU,omitempty"`
+		Envs   []struct {
+			Name  string `yaml:"Name,omitempty"`
+			Value string `yaml:"Value,omitempty"`
+		} `yaml:"Envs,omitempty"`
+		URL []struct {
+			// Name of the URL
+			Name string `yaml:"Name,omitempty"`
+			// Port number for the url of the component, required in case of components which expose more than one service port
+			Port int `yaml:"Port,omitempty"`
+		} `yaml:"Url,omitempty"`
+	} `yaml:"ComponentSettings,omitempty"`
+}
+
+// VerifyLocalConfig verifies the content of the config.yaml file
+func verifyLocalConfig(context string) config {
+	var conf config
+
+	yamlFile, err := ioutil.ReadFile(context)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
why is it just printing the error?

Reply:
> why is it just printing the error?

Config.yaml file will exist anyway. So it is just for validation of ReadFile. As such most likely it will read the file there is no confusion, so just in case it happens which is probably not going to happen. I just printed the error.

Reply:
Use `glog.V(4).Info` for such use cases